spring-cloud-dataflow-docs/src/main/asciidoc/streams.adoc

```diff
@@ -138,3 +138,48 @@ When a stream is comprised of multiple modules with the same name, they must be
 
   stream create --definition "http | firstLabel: transform --expression=payload.toUpperCase() | secondLabel: transform --expression=payload+'!' | log" --name myStreamWithLabels --deploy
 
+[[tap-dsl]]
+== Tap DSL
+
+Taps can be created at various producer endpoints in a stream. For a stream like this:
+
+  stream create --definition "http | step1: transform --expression=payload.toUpperCase() | step2: transform --expression=payload+'!' | log" --name mainstream --deploy
+
+taps can be created at the output of `http`, `step1` and `step2`.
+
+Tap DSL requires to specify the `source destination name` for the tap stream. The syntax for source destination name is:
```

Comment:
"Tap DSL" sounds too formal, since now we are simply referring to a destination (as with any use of destination, not just "taps") while we happen to know the naming convention that is used for the destinations created on behalf of streams. I would change this to something like: "To create a stream that acts as a 'tap' on another stream, specify..."